actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -1072,15 +1076,15 @@ public void dispose()
             @Override
             public void run()
             {
-                if (localActor.isDeactivated())
+                if (localActor.isDeactivated() || getState() != NodeCapabilities.NodeState.RUNNING)
```

Comment:
NodeCapabilities.NodeState.RUNNING is set before the node joins the cluster inside cloud.orbit.actors.Stage#start and at that point should not receive any requests until that future completes.

If the node is shutting down I don't think it makes sense to continue running the timer task if it hasn't passed this checkpoint.

I'm happy to revert these two changes to the TimerTask and open a new pull request as they are not 100% related to the lock on shutdown.